logo-ann/generation/01_generate_image_dump.py

```diff
@@ -160,8 +167,10 @@ def split_barcode(barcode: str) -> List[str]:
     return [barcode]
 
 
-def generate_image_path(barcode: str, image_id: str) -> str:
+def generate_image_path(barcode: str, image_id: str) -> Optional[str]:
     splitted_barcode = split_barcode(barcode)
+    if splitted_barcode == None:
```

Comment:
You should use `is` when checking if a value is None: `splitted_barcode is None`